### PR TITLE
The initial README for scoverage

### PR DESCRIPTION
**Pull request by rolandtritsch**

### CODE REVIEW

**Comment by rolandtritsch** [comment]
Hi @ckipp01. Hi @sksamuel. 

Let's use this PR to develop a first version of the scoverage README (to replace the scoverage web-site).

You are both admins on this repo. Who else needs access to this repo? Who else needs to be part of this disussion?

**Comment by rolandtritsch** [comment]
Proposed high-level structure ...

* Introduction to scoverage (all of the repos, functions and features)
* Getting started (just pointing/linking to existing documentation in the repos)
* How to contribute (high level agreement, but then pointing to the `CONTRIBUTING.md` files in the repos)

**Comment by rolandtritsch** [comment]
Hi @ckipp01. Hi @sksamuel,

Was also wondering, if we should setup a gitter/discord/slack workspace to streamline communication for the people that want to work on the project (I have one more person who would be interested to help). 

WDYT?

**Comment by sksamuel** [comment]
I like what you're working on.
I don't work with Scala anymore, so would happy to hand this project off to another keen Scala developer(s). It's also great that Scala3 has added scoverage support directly into the compiler. Would you be interested in taking ownership?

File: profile/README.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
-# SCoverage - Scala Code Coverage Tools
+# scoverage - Scala Code Coverage Tools
 
-Welcome to `SCoverage`, a collection of tools to measure code coverage for [Scala][] code.
+![report][]
 
-[Scala]: https://www.scala-lang.org/
+Welcome to `scoverage`, a collection of tools to measure code coverage for [scala][] code.
```
Comment (ckipp01) [inline profile/README.md]:
```suggestion
Welcome to `scoverage`, a collection of tools to measure code coverage for [Scala][] code.
```

File: profile/README.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+
+`scoverage` features/repositories ...
+
+- [scalac-scoverage-plugin][] - the compiler plugin for Scala 2 and lots of other utilities (e.g. the report writers) that are used by the tools
+- [sbt-scoverage][] - the [sbt][] plugin
+- [gradle-scoverage][] - the [gradle][] plugin
+- [scoverage-maven-plugin][] - the [maven][]
+- [sbt-coveralls][] - the sbt plugin to upload scoverage reports to [coveralls][]
+- [sbt-scoverage-examples][] - example reports, which show how to use scoverage
+
```
Comment (ckipp01) [inline profile/README.md:22]: I know this is all the repos contained in the org, but do you think we should also find a way to link to the Mill support, which is directly in Mill?

**Review by ckipp01 — APPROVED** [review]
> Proposed high-level structure ...

I think this is all good.

> Was also wondering, if we should setup a gitter/discord/slack workspace to streamline communication for the people that want to work on the project (I have one more person who would be interested to help).

There already is a gitter channel. It can be bridged with Matrix as well, which is how I field questions on there. I'm not sure I'd recommend setting up a new channel yet until there is more activity there. I think simply watching the current channel and making it more "alive" would be better.

File: profile/README.md (hunk 1 above)
Comment (rolandtritsch) [reply profile/README.md]: Fixed.

File: profile/README.md (hunk 2 above)
Comment (rolandtritsch) [reply profile/README.md:22]: Yes. Good point. Done.

**Comment by rolandtritsch** [comment]
> I like what you're working on. I don't work with Scala anymore, so would happy to hand this project off to another keen Scala developer(s). It's also great that Scala3 has added scoverage support directly into the compiler. Would you be interested in taking ownership?

Hi @sksamuel. Yes. Can you make @ckipp01 and myself owners for `scoverage`.

**Comment by sksamuel** [comment]
Done, you are both now co-owners of the org.

**Pull request merged**